code/src/components/RemoveTasks.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { tasks } from 'reducers/tasks'
+
+
+export const RemoveTasks = () => {
+
+  const items = useSelector((store) => store.tasks.items);
+  const enabled = items.length > 0;
```

Comment:
Was a bit unsure what this was until I found the variable in the buttons. Smart way of setting the buttons to disabled. However, would have been nice with either a more descriptive name like enableButtons or similar, or a comment above 😊